components/network/megasync/Makefile

```diff
@@ -49,19 +49,28 @@ COMPONENT_SRC_1=		sdk-$(GIT_BRANCH_1)
 COMPONENT_ARCHIVE_1=      $(COMPONENT_SRC_1).tar.gz
 COMPONENT_ARCHIVE_URL_1=  $(COMPONENT_GIT_URL)/sdk/archive/$(GIT_BRANCH_1).tar.gz
 COMPONENT_ARCHIVE_HASH_1= sha256:850973860197be5328324970572844fbd833439e65f14e23b78fd15a1c8a6fad
+COMPONENT_PKGLINT_ENV += ENCUMBERED=true
 
```

Comment:
It is not allowed for packages outside encumbered to depend on encumbered packages.
This is due to legal requirements.
Either drop the dependency on ffmpeg or move this package to encumbered (obsoleting network/megasync)!

Reply:
Can I move the megasync folder to encumbered, change it to _include $(WS_MAKE_RULES)/encumbered.mk_ and reuse network/megasync as fmri (like media/vlc in vlc)?